Makefile

```diff
@@ -23,27 +24,25 @@ clean-pyc:
 	find . -name '__pycache__' -exec rm -rf {} +
 
 lint:
-	tox run -e lint
-
-lint-roll:
-	isort eth tests scripts
-	black eth tests scripts setup.py
-	$(MAKE) lint
+	@pre-commit run --all-files --show-diff-on-failure || ( \
+		echo "\n\n\n * pre-commit should have fixed the errors above. Running again to make sure everything is good..." \
+		&& pre-commit run --all-files --show-diff-on-failure \
+	)
 
 test:
-	py.test --tb native tests
-
-test-all:
-	tox
+	pytest tests
 
 coverage:
 	coverage run --source eth
 	coverage report -m
 	coverage html
 	open htmlcov/index.html
 
-build-docs: clean
-	cd docs/; sphinx-build -W -T -E . _build/html
+build-docs:
```

Comment:
This may also be user error/not having the right things installed, but I'm seeing a bunch of warnings when I run `make build-docs`. For example:
- `WARNING: autodoc: failed to import module 'run' from module 'scripts.benchmark'; the following exception was raised:                                                                 No module named 'checks'`
- `~/ef/py-evm/docs/tests.rst: WARNING: document isn't included in any toctree`
And:
```
Document: guides/building_an_app_that_uses_pyevm
------------------------------------------------
**********************************************************************
File "guides/building_an_app_that_uses_pyevm.rst", line 100, in default
Failed example:
    print(f"The balance of address {encode_hex(MOCK_ADDRESS)} is {mock_address_balance} wei"
Exception raised:
    Traceback (most recent call last):
      File "/opt/homebrew/Cellar/python@3.11/3.11.4_1/Frameworks/Python.framework/Versions/3.11/lib/python3.11/doctest.py", line 1351, in __run
        exec(compile(example.source, filename, "single",
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
      File "/Users/eve/ef/py-evm/venv-upgrade-template/lib/python3.11/site-packages/sphinx/ext/doctest.py", line 484, in compile
        return compile(code, name, self.type, flags, dont_inherit)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
      File "<doctest default[10]>", line 1
        print(f"The balance of address {encode_hex(MOCK_ADDRESS)} is {mock_address_balance} wei"
             ^
    SyntaxError: '(' was never closed
**********************************************************************
```

Are you seeing those too?

Reply:
I get the same errors, which makes me wonder why CI docs tests are passing. Looking into.

Reply:
Once tests got running again, this turned out to indeed be a `(` I didn't close.